docs/machine-learning/tutorials/health-violation-classification-model-builder.md

```diff
@@ -0,0 +1,169 @@
+
+![Model Builder wizard in Visual Studio](./media/sentiment-analysis-model-builder/model-builder-screen.png)
+
+To train your model, you need to select from the list of available machine learning scenarios provided by Model Builder. In this case, the scenario is `Issue Classification`.
+
+1. In **Solution Explorer**, right-click the *RestaurantViolations* project, and select **Add** > **Machine Learning**.
+1. For this sample, the scenario is sentiment analysis. In the *scenario* step of the Model Builder tool, select the **Issue Classification** scenario.
+
+## Load the data
+
+Model Builder accepts data from two sources, a SQL Server database or a local file in `csv` or `tsv` format.
+
+1. In the data step of the Model Builder tool, select **SQL Server** from the data source dropdown.
+1. Select the button next to the **Connect to SQL Server database** text box.
+    1. In the **Choose Data** dialog, select **Microsoft SQL Server Database File**. 
+    1. Uncheck the **Always use this selection** checkbox and select **Continue**
+    1. In the **Connection Properties** dialog, select **Browse** and select the downloaded *RestaurantScores.mdf* file.
+    1. Select **OK**
+1. Choose **Violations** from the **Table Name** dropdown.
+1. Choose **RiskCategory** in the **Column to Predict (Label)** dropdown.
+1. Leave the default column selections, **InspectionType** and **ViolationDescription**, checked in the **Input Columns (Features)** dropdown.
+1. Select the **Train** link to move to the next step in the Model Builder tool.
+
+## Train the model
+
+The machine learning task used to train the issue classification model in this tutorial is multiclass classification. During the model training process, Model Builder trains separate models using different multiclass classification algorithms and settings to find the best performing model for your dataset.
+
+The time required for the model to train is proportional to the amount of data. Model Builder automatically selects a default value for **Time to train (seconds)** based on the size of your data source.
+
+1. Although Model Builder sets the value of **Time to train (seconds)** to 10 seconds, increase it to 30 seconds. Training for a longer period of time allows Model Builder to explore a larger number of algorithms and combination of parameters in search of the best model.
+1. Select **Start Training**.
+
+    Throughout the training process, progress data is displayed in the `Progress` section of the train step.
+
+    - Status displays the completion status of the training process.
+    - Best accuracy displays the accuracy of the best performing model found by Model Builder so far. Higher accuracy means the model predicted more correctly on test data.
+    - Best algorithm displays the name of the best performing algorithm performed found by Model Builder so far.
+    - Last algorithm displays the name of the algorithm most recently used by Model Builder to train the model.
```

Comment:
```suggestion
    - **Last algorithm** displays the name of the algorithm most recently used by Model Builder to train the model.
```

Reply:
Note sure whether the items I bolded are code or part of the UI. If code or log output, please format with `<term>`.